skyportal/handlers/api/spectrum.py

```diff
@@ -635,6 +653,10 @@ def get(self, obj_id):
                 key=lambda x: x["created_at"],
                 reverse=True,
             )
+            annotations = [
+                {**a.to_dict(), 'author': a.author.to_dict()} for a in annotations
```

Comment:
Why is this necessary?

Reply:
I think without this the annotations do not come with an author (which is displayed in the table).

Reply:
Nevermind it works without this. I don't know why but for comments this was needed (maybe there it required the gravator or something).

Reply:
I don't know why we need this here and not in the object annotations. But it doesn't render an author if I don't add this...

Reply:
Okay, this is okay to leave as is then. Thanks